index.html

```diff
@@ -187,6 +187,39 @@ <h2>Skills</h2>
                     </ul>
                 </section>
             </section>
+            <section class="form-container">
+                <div class="form-title">
+                    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in</h3>
+                </div>
+                <form action="https://formspree.io/f/xayajwko" method="POST">
+                    <ul>
+                        <li class="name">
+                            <label for="name"></label>
+                            <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                        </li>
+                        <li class="name">
+                            <label for="mail"></label>
+                            <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                        </li>
+                        <li class="textarea">
+                            <label for="msg"></label>
+                            <textarea id="msg" name="user_message" placeholder="Enter text here" maxlength="500" required></textarea>
```

Comment:
`<texarea>` div should have a `type="textarea"` tag. This is easy to do, please add the tag. 🙏

Reply:
I'm not sure but <textarea> won't accept the type attribute. Can you provide more details, please?

Reply:
You're right, using `<textarea>` is enough to have this point valid (without type). Just continue to the next step 🐶